firebase-vscode/webviews/globals/html-broker.ts

```diff
@@ -1,22 +1,72 @@
+import { useEffect, useState } from "react";
 import { Broker, createBroker } from "../../common/messaging/broker";
 import {
   ExtensionToWebviewParamsMap,
   WebviewToExtensionParamsMap,
 } from "../../common/messaging/protocol";
 
+export function useBrokerListener<
```

Comment:
These were already added in the private repository but I forgot to sync those here.